_Sidebar.md

```diff
@@ -0,0 +1,15 @@
+- [Guide for å bidra til LKKs oppgavesamling](./Guide-for-å-bidra-til-LKKs-oppgavesamlingt)
```

Comment:
`(./Guide-for-å-bidra-til-LKKs-oppgavesammling)`

Fjern den eine m-en når du rettar opp.